### PR TITLE
Get metadata for phpfpm master and workers process 

### PR DESCRIPTION
**Pull request by achawla2012**
### Proposed changes

 This change determines metadata for php-fpm master and worker metadata with focuss on parity/functionality with old agent)

### Testing
Unit tests/shell commands execution. Future:- will leverage `gopsutil` package for `shell` command execution.

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [X] I have read the [`CONTRIBUTING`](https://github.com/nginx/agent/blob/main/docs/CONTRIBUTING.md) document
- [X] I have run ```make install-tools``` and have attached any dependency changes to this pull request
- [X] If applicable, I have added tests that prove my fix is effective or that my feature works
- [X] If applicable, I have checked that any relevant tests pass after adding my changes
- [ ] If applicable, I have updated any relevant documentation ([`README.md`](https://github.com/nginx/agent/blob/main/README.md))
- [ ] If applicable, I have tested my cross-platform changes on Ubuntu 22, Redhat 8, SUSE 15 and FreeBSD 13

### CODE REVIEW

File: src/core/nginx.go [hunk 1]
```diff
@@ -154,7 +154,7 @@ func (n *NginxBinaryType) GetNginxIDForProcess(nginxProcess *Process) string {
 }
 
 func (n *NginxBinaryType) getNginxIDFromProcessInfo(nginxProcess *Process, info *nginxInfo) string {
-	return GenerateNginxID("%s_%s_%s", nginxProcess.Path, info.confPath, info.prefix)
+	return GenerateID("%s_%s_%s", nginxProcess.Path, info.confPath, info.prefix)
```
Comment (achawla2012) [inline src/core/nginx.go:157]: using this to generate local_id for php process. So, is this fine ?

File: src/extensions/php-fpm-metrics/master/manager.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package master
```
Comment (oliveromahony) [inline src/extensions/php-fpm-metrics/master/manager.go:1]: suggest revisiting the master naming to be more something else? manager?

File: src/extensions/php-fpm-metrics/pool/worker/worker.go [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+		if len(directive) != 2 {
+			continue
+		}
+		if directive[0] == "include" {
+			// found an include
+			currentPool.Includes = getIncludes(w.dir, directive[1])
+		} else if directive[0] == "listen" {
+			// found listen socket
+			currentPool.Listen = directive[1]
+			currentPool.Flisten = getFlisten(currentPool, directive[1], line)
```
Comment (dhurley) [inline src/extensions/php-fpm-metrics/pool/worker/worker.go:70]: Can you explain what `Flisten` is?

**Review by dhurley — COMMENTED** [review]
The formatting doesn't look right in some files. Could you run `make format`?

**Pull request closed without merging**